tests/storage_tests.cpp

```diff
@@ -4,6 +4,26 @@
 
 using namespace sqlite_orm;
 
+#ifdef SQLITE_ORM_CTAD_SUPPORTED
+TEST_CASE("connection control") {
+    const auto openForever = GENERATE(false, true);
```

Comment:
what is `GENERATE`?

Reply:
A Catch2 [data generator](https://github.com/catchorg/Catch2/blob/devel/docs/generators.md).